resources/styles/variables.less

```diff
@@ -32,6 +32,8 @@
 
 @reading-color: @tutor-quaternary;
 @homework-color: @tutor-tertiary-light;
+@practice-color:  darken(@homework-color, 10%);
```

Comment:
could these be prefixed with "qa-"? then, if they get used elsewhere we can drop the prefix